Modules/timemodule.c

```diff
@@ -1485,7 +1485,8 @@ _PyTime_GetThreadTimeWithInfo(PyTime_t *tp, _Py_clock_info_t *info)
 
 #elif defined(HAVE_CLOCK_GETTIME) && \
```

Comment:
Please add a comment to explain why the clock is disabled on NetBSD, with a reference to the issue (gh-123978).